src/main/java/frc/robot/RobotContainer.java

```diff
@@ -230,46 +231,46 @@ private void configureBindings() {
         primaryXboxController.a().whileTrue(algaeAlignPPOTF.algaeAlign());
         primaryXboxController.button(1).whileTrue(reefAlignPPOTF.reefAlign());
         gigaStation
-                .button(2)
+                .topMiddleSwitch()
                 .onTrue(
                         coralManipulator
                                 .transitionTo(CoralManipulatorState.CLIMB)
                                 .alongWith(leds.runPattern(LEDPatterns.FADING_BLUE_SCROLL)));
-        gigaStation.button(18).onTrue(coralManipulator.transitionTo(CoralManipulatorState.STOWED));
-        gigaStation.button(16).onTrue(coralManipulator.grabber.transitionTo(GrabberState.OFF));
+        gigaStation.gr2().onTrue(coralManipulator.transitionTo(CoralManipulatorState.STOWED));
+        gigaStation.wr2().onTrue(coralManipulator.grabber.transitionTo(GrabberState.OFF));
         gigaStation
-                .button(17)
+                .gl2()
                 .onTrue(coralManipulator.grabber.transitionTo(GrabberState.ALGAE_SHOOT));
-        gigaStation.button(7).onTrue(coralManipulator.setQueueState(CoralManipulatorState.L1));
-        gigaStation.button(8).onTrue(coralManipulator.setQueueState(CoralManipulatorState.L2));
-        gigaStation.button(9).onTrue(coralManipulator.setQueueState(CoralManipulatorState.L3));
-        gigaStation.button(10).onTrue(coralManipulator.setQueueState(CoralManipulatorState.L4));
+        gigaStation.bl1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L1));
+        gigaStation.br1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L2));
+        gigaStation.wl1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L3));
+        gigaStation.wr1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L4));
```

Comment:
can you make the names of the functions more descriptive? it should be readable and clear to someone who is not familiar with the shorthand we use. something like: 
```suggestion
        gigaStation.blackLeft1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L1));
        gigaStation.blackRight1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L2));
        gigaStation.whiteLeft1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L3));
        gigaStation.whiteRight1().onTrue(coralManipulator.setQueueState(CoralManipulatorState.L4));
```

tbh i feel like the naming scheme isn't very clear either to someone who didn't design this

![6C62EBF7-7B3E-4DD4-BD7E-EC568DC58A25](https://github.com/user-attachments/assets/f9d6abcb-1703-4d6d-82ac-06db3e9c7502)